### ISSUE TITLE
Requests for tvOS 11.3

### ISSUE THREAD

**Issue by rat-moonshine** [filsv/iPhoneOSDeviceSupport#9]
Can you please include support files for tvOS, too? 

Thanks!

**Comment by filsv** [comment]
Yup, tomorrow it's will be available here, and you can download it - https://github.com/filsv/TVOSDeviceSupport ;)
